mantle/platform/qemu.go

```diff
@@ -577,22 +531,44 @@ func (builder *QemuBuilder) addDiskImpl(disk *Disk, primary bool) error {
 		return err
 	}
 
+	// MultiPathDisks must be NBD remote mounted
+	if disk.MultiPathDisk || disk.NbdDisk {
+		socketName := fmt.Sprintf("%s.socket", dstFileName)
+		disk.attachEndPoint = fmt.Sprintf("nbd:unix:%s", socketName)
```

Comment:
Are we just doing these sockets in the current directory?  Are they abstract or unix domain?  We should probably be using a temporary path if the latter.

Reply:
I was basing it off the `dstFileName` which is generated from `mktemp`. That said, I've added some handling for tmpfiles.